### DISCUSSION TITLE
No inquiry mode

### DISCUSSION THREAD

**Discussion by robertourio** [darthcloud/BlueRetro#559] [Q&A] [answered]
Hi 
I was trying to pair some bluetooth controller but my device never begin the inquiry mode.
No blinks the led of pairing even if I pulse de Boot button.
I have others device that work perfectly but not this last one.
Usually if the devices works fine they write many log messages while they are on inquiry mode, but  mine write the log that I attach, and write no more.
Any help will be thanked.
Thanks
[mal.txt](https://github.com/darthcloud/BlueRetro/files/10439626/mal.txt)

**Comment by darthcloud** [comment, marked as answer]
Use HW1 firmware

**Reply by robertourio** [reply]
Thanks. It works!!
